CMakeLists.txt

```diff
@@ -127,13 +127,29 @@ ELSE ()
 ENDIF()
 
 if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
-    set(BOOST_HINTS "C:\\\\Boost")
-endif ()
+	set(BOOST_HINTS "C:\\\\Boost")
+
+	if (${CMAKE_BUILD_TYPE} MATCHES "Release")
+		set(Boost_USE_STATIC_LIBS        ON)  # only find static libs
+		set(Boost_USE_MULTITHREADED      ON)
+		set(Boost_USE_STATIC_RUNTIME     OFF)
+		set(Boost_USE_DEBUG_RUNTIME      OFF)
+
+		set(Boost_USE_DEBUG_LIBS         OFF) # ignore debug libs and
+		set(Boost_USE_RELEASE_LIBS       ON)  # only find release libs
+	else()
+		set(Boost_USE_DEBUG_LIBS         ON) # ignore debug libs and
```

Comment:
The comment has an error. It says the same as the other branch.

Reply:
fixed.